maestro/tools/check-schemas.sh

```diff
@@ -15,6 +15,7 @@ AGENT_FILES=$(find . -name '*agents*.yaml')
 echo "|Filename|Type|Stats|" >> "$GITHUB_STEP_SUMMARY"
 echo "|---|---|---|" >> "$GITHUB_STEP_SUMMARY"
 
+EXCLUDED_FILES=("./crewai_test/src/crewai_test/config/agents.yaml" "./demos/agents/crewai/generic/config/agents.yaml" "./demos/agents/crewai/activity_planner/config/agents.yaml")
```

Comment:
Why don't we call these `config` instead of `agent` since that's what they are parts of the agent configuration?

Reply:
The file name `agent.yaml` seems hard coded in the crewai framework or SDK.  The other name doesn't work.